modules/schemas/src/main/resources/contexts/nexus/core/resource/v0.2.0.json

```diff
@@ -0,0 +1,12 @@
+{
+  "@context": {
+    "type": "@type",
+    "id": "@id",
+    "distribution": {
+      "@id": "http://schema.org/distribution"
+    },
+    "links": {
+      "@id": "https://bbp-nexus.epfl.ch/vocabs/nexus/core/terms/v0.2.0/links"
```

Comment:
I would keep this term version to v0.1.0. The context changed not the nxv:links term